types/ember-cli-fastboot/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for ember-cli-fastboot 2.2
+// Project: https://github.com/ember-fastboot/ember-cli-fastboot
+// Definitions by: Henry Majoros <https://github.com/hmajoros>
+//                 Dan Freeman <https://github.com/dfreeman>
+//                 Chris Krycho <https://github.com/chriskrycho>
+//                 James C. Davis <https://github.com/jamescdavis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
```

Comment:
```ts
// TypeScript Version: 4.4
```
why this version? `@ember/service` seems to require 3.7

Reply:
There's an in-progress PR updating all the Ember types to require 4.4 with the Ember v4 types. This aligns with the TS support policy Ember will most likely be adapting (currently in the RFC process).

Reply:
ok, thx for explanation!